README.md

```diff
@@ -84,3 +84,15 @@ on initial render. Receives two arguments: `keypath` and `value`. Defaults to
 Optional parameters for filterer (search). Must be an object.
 
 - `ignoreCase`, Set to `true` to enable case insensitivity in search. Defaults to `false`.
+
+#### props.rootExpanded
+
+Optional boolean to expand the root object. Defaults to true.
```

Comment:
Not sure about this option, since there already exists a `isExpanded` predicate, I would like to rework it to be applied to root node as well. What do you think?